modules/oc-mirror-v2-support.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+// * installing/disconnected_install/installing-oc-mirror-v2.adoc
+// * updating/updating_a_cluster/updating_disconnected_cluster/mirroring-image-repository.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="oc-mirror-v2-support_{context}"]
+= = oc-mirror V2 compatibility and support
```

Comment:
```suggestion
= oc-mirror V2 compatibility and support
```
Not resolving in the preview because we have the extra = https://76917--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/disconnected_install/about-installing-oc-mirror-v2.html#oc-mirror-v2-support_installing-mirroring-disconnected